src/PKSim.Presentation/Presenters/Charts/ChartPresenter.cs

```diff
@@ -242,24 +194,19 @@ protected virtual void OnDragOver(object sender, DragEventArgs e)
 
       protected virtual void OnDragDrop(object sender, DragEventArgs e)
       {
-         var droppedObservedData = _observedDataDragDropBinder.DroppedObservedDataFrom(e);
-         droppedObservedData.Each(data => AddObservedData(data, asResultOfDragAndDrop: true));
+         var droppedObservedData = _observedDataDragDropBinder.DroppedObservedDataFrom(e).ToList();
+         AddObservedData(droppedObservedData, asResultOfDragAndDrop: true);
       }
 
       protected override void NotifyProjectChanged()
       {
          _chartTask.ProjectChanged();
       }
 
-      private void setDefaultEditorLayout()
+      protected override void ConfigureColumns()
```

Comment:
This is the only methods. Called ConfigureColumns, allow the app to override the default behavior